packages/client/src/pages/Map/Map.tsx

```diff
@@ -90,15 +114,21 @@ class Map extends React.Component<MapProps> {
 
     private onTrecipeCopyClick() {
         if (this.props.trecipe) {
-            this.props.duplicateTrecipe(this.props.trecipe.uuid);
-            // TODO: Redirect to Map page of copied Trecipe
+            this.props.duplicateTrecipe(this.props.trecipe.uuid, (uuid: string) => {
+                // TODO: update this when route is updated in a future PR
+                // Redirect to Trecipe map page
+                this.props.history.push(`/map/${uuid}`);
+            });
```

Comment:
Hmmm I am not sure about that, I think BrowserRouter has its own copy of history. Dont know if something outside of components can access the history 
And yea, it would be great if we can access history in action so we wont have to do this clunky callback to navigate pages lol